Configuration/PyReleaseValidation/python/relval_2017.py

```diff
@@ -48,17 +48,17 @@
            10824.6,11024.6,11224.6,
            10824.8,
            11024.15,
-           10842.501,10842.502, # 10842.503,10842.504,
-           10824.501,10824.502, # 10824.503,10824.504,
-           10824.511,10824.512, # 10824.513,10824.514,
-           10824.521,10824.522, # 10824.523,10824.524,
+           10842.501, # 10842.503,10842.504,
+           10824.501, # 10824.503,10824.504,
+           10824.511, # 10824.513,10824.514,
+           10824.521, # 10824.523,10824.524,
```

Comment:
could you remove also the (commented out) `.503`, `.513`, `.523` etc. workflows ?
those will (eventually) run only on a GPU machine

Reply:
(here and in the other blocks, as well)